Dockerfile

```diff
@@ -7,37 +7,30 @@ EXPOSE 8182
 RUN yum -y install epel-release &&\
     yum -y install git zip unzip awscli &&\
     yum -y install java java-devel maven &&\
+    yum -y install wget &&\
     yum clean all
 
+
 ENV JAVA_HOME /usr/lib/jvm/java-openjdk
 ENV M2_DIR=/m2
 ENV M2_REPO=${M2_DIR}/repository
 ENV MAVEN_OPTS="-Dmaven.repo.local=${M2_REPO}"
 
-RUN curl -o /opt/titan-1.1.0-SNAPSHOT-hadoop2.zip https://s3.amazonaws.com/gremlin-tarballs/titan-1.1.0-SNAPSHOT-hadoop2.zip
-RUN curl -o /tmp/titan-repo.tgz https://s3.amazonaws.com/gremlin-tarballs/titan-all.tgz &&\
-    mkdir -p ${M2_DIR} &&\
-    tar xzf /tmp/titan-repo.tgz -C ${M2_DIR} &&\
-    mv ${M2_DIR}/root/.m2/repository/ ${M2_DIR} &&\
-    rm -Rf ${M2_DIR}/root/ /tmp/titan-repo.tgz
-
-# Prep, build and install DynamoDB storage backend driver with support for titan, and install Gremlin server
-RUN git clone https://github.com/awslabs/dynamodb-titan-storage-backend.git /opt/dynamodb/dynamodb-titan-storage-backend/ &&\
-    cd /opt/dynamodb/dynamodb-titan-storage-backend/ &&\
-    git checkout d1a59624dcef796b835e7ffb41f0a3f007008d63 -b "last-working" &&\
-    curl https://gist.githubusercontent.com/pluradj/d56c1948f4665ee7fb1bc35daeba4f92/raw/be5f639a64c8d6ac196c59eb7e6d1a1903015b17/dynamo-titan11-tp323.patch | git apply -v --index &&\
-    mvn clean install
 
+RUN git clone https://github.com/awslabs/dynamodb-janusgraph-storage-backend.git /opt/dynamodb/dynamodb-janusgraph-storage-backend/ &&\
```

Comment:
If we want to build the JARs ourselves then we should probably be more specific here and build from a specific commit hash. Otherwise every build will just take what's currently in master branch -- including bug fixes, but also new features and possibly non-backward compatible changes.

Reply:
But i thought thats the idea of moving to Janus from titan that we wanted latest updates. If stable version was all we wanted, even titan was providing that. please correct me if im wrong here.

Reply:
That's not entirely true @yzainee . Titan is no longer supported. Stable is one part, support is another. Hence this move to JanusGraph. Again, we should make sure that we have stable production builds rather than latest and greatest because that might break somethings.

Reply:
Now taking the latest version --branch jg0.2.0-1.2.0